<!--
-->
<launch>

  <arg name="bag_path" default="$(env HOME)/.ob/bag"/>
  <arg name="bag_name" default="ob_record_data"/>

  <param name="/use_sim_time" value="true" />

  <node pkg="rosbag" type="play" name="playbag" output="log" respawn="false" args="$(arg bag_path)/$(arg bag_name).bag --clock --topic /scan /tf
"/>

  <include file="$(find obrobot_navigation)/launch/include/odom/gmapping_base_offline.launch"/>

  <!--<node pkg="rviz" type="rviz" name="runrviz" output="screen" respawn="false" />-->

  <!--<include file="$(find obrobot_description)/launch/obrobot_description.launch"/>-->
  <!--<include file="$(find obrobot_navigation)/launch/include/odom/gmapping_base_offline.launch"/>-->
  <!--<include file="$(find obrobot_navigation)/launch/include/odom/teb_move_base.launch"/>-->
<!--
  <include file="$(find world_canvas_server)/launch/world_canvas_server.launch"/>
  <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"/>
-->
  <!--<node name="robot_pose_publisher" pkg="robot_pose_publisher" type="robot_pose_publisher" />
  <node pkg="laser_filters" type="scan_to_scan_filter_chain" output="screen" name="laser_filter">
      <rosparam command="load" file="$(find obrobot_tools)/conf/box_filter.yaml" />
  </node>-->

<!--
  <node name="robot_data" pkg="obrobot_tools" type="robot_data.py" respawn="true" />
  <node name="cmd_listen" pkg="obrobot_tools" type="cmdListen.py" respawn="true" />
-->

<!--
  <node name="auto_map_find_goal" pkg="turtlebot_navigation" type="ob_img_1207.py" respawn="true" />
  <node name="auto_map_nav" pkg="turtlebot_navigation" type="ob_nav.py" respawn="true" />
-->

  <!-- rviz driver -->
  <!--<include file="$(find obrobot_rviz)/launch/view_navigation.launch" />-->

</launch>



